test/documentation/DoxygenLayout.xml

```diff
@@ -0,0 +1,206 @@
+<doxygenlayout version="1.0">
+  <!-- Generated by doxygen 1.8.14 -->
+  <!-- Navigation index tabs for HTML output -->
+  <navindex>
+    <tab type="mainpage" visible="yes" title=""/>
+<!--     <tab type="pages" visible="yes" title="" intro=""/> -->
+
+    <tab type="user" visible="yes" title="Setup" url="@ref setup" intro=""/>
+    <tab type="usergroup" visible="yes" title="Tutorial" intro="">
+        <tab type="user" visible="yes" title="A first example" url="@ref tutorial_first_example" intro=""/>
+        <tab type="user" visible="yes" title="Alphabets" url="@ref tutorial_alphabets" intro=""/>
+    </tab>
+    <tab type="usergroup" visible="yes" title="Notes" intro="">
+        <tab type="user" visible="yes" title="API stability" url="@ref notes_stability" intro=""/>
+        <tab type="user" visible="yes" title="Serialisation" url="@ref notes_serialisation" intro=""/>
+    </tab>
+    <tab type="modules" visible="yes" title="API Reference (Modules)" intro=""/>
+    <tab type="usergroup" visible="yes" title="API Reference (details)" intro="">
+        <tab type="namespaces" visible="yes" title="">
+        <tab type="namespacelist" visible="yes" title="" intro=""/>
```

Comment:
indent one more? :nail_care:

Reply:
to me this seems fine, always 4 space indentation...

Reply:
I meant four more since `namespacelist` is inside of `namespace` :nerd_face:

Reply:
but its not :wink

Reply:
but namespaces is a `<tab ...>` and not `<tab .../>`. I thought the former open a new layer and is closed later?

Reply:
@smehringer is right. the inner tabs need to be indented to match the indentation style above. To me it seems that the `<tab>... </tab>` is an enclosing block, so why would you not indent the lower levels there?

Reply:
@smehringer you are right